### PR TITLE
Resolve: updating to v18 compatibility

### PR DESCRIPTION
**Pull request by jakubjezek001**
## Brief description
Resolve is now supported up to v18

## Description
Latest version v18 is supporting Python installation for higher then 3.6. We have been testing it on version 3.9.13. OpenTimelineIO is working without glitch at version 0.15.  

## Additional info
- removed lot of unused code
- settings were reduced because most of the paths could be moved to pre launch hool

## Testing notes:
1. Install BlackMagic Davinci Resolve v18
2. follow instructions form `openpype\hosts\resolve\README.markdown` for installation of OpenTimelineIO and PySide2
3. Start OpenPype and Launch Resolve from any project it had been assign to.
5. Try Create, Publish, Load, Manager

### CODE REVIEW

**Comment by jakubjezek001** [comment]
resolve https://github.com/pypeclub/OpenPype/issues/3911

File: openpype/hosts/resolve/hooks/pre_resolve_setup.py [hunk 1]
```diff
@@ -14,35 +13,91 @@ class ResolvePrelaunch(PreLaunchHook):
     app_groups = ["resolve"]
 
     def execute(self):
+        current_platform = platform.system().lower()
+
+        PROGRAMDATA = self.launch_context.env["PROGRAMDATA"]
```
Comment (iLLiCiTiT) [inline openpype/hosts/resolve/hooks/pre_resolve_setup.py:18]: This will crash on linux or mac because they don't have `PROGRAMDATA`.

Comment (jakubjezek001) [reply openpype/hosts/resolve/hooks/pre_resolve_setup.py:18]: good catch ;)

**Comment by jrsndl** [comment]
I was able to launch Resolve 18 with Python 3.9.13, windows 10.
Loading Publishing and Manager works. Conform lock enabled has to be off for properly switching versions in the timeline.
Loading exr loads all representations to the Media tab.

**Review by jrsndl — COMMENTED** [review]
I was able to launch Resolve 18 with Python 3.9.13, windows 10.
Loading Publishing and Manager works. Conform lock enabled has to be off for properly switching versions in the timeline.
Loading exr loads all representations to the Media tab.

**Review by jrsndl — APPROVED** [review]

**Pull request merged**